test/functionalTest/cases/3162_context_providers_legacy_non_primitive/registration_without_provider.test

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+BUG 
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create entity E1 without attributes
+# 02. Append new geo:box attribute (attr3) -> OK
+#
```

Comment:
A step 03 with GET to check that the attributes has been really appended should be included.

Reply:
Need to take a look at this test to understand what has happened.
The way it is ... makes no sense whatsoever.

The idea behind step 2 is to actually **find** the entity (entities without attr that have a registration weren't found).
A recent PR of mine fixes this.
But, I found that bug myself implementing the fix of this PR

Reply:
Si, I looked into this a little ...
This test I used simply to provoke the bug fixed in https://github.com/telefonicaid/fiware-orion/pull/3193.
its current contents is what remains after tests, with the registration removed.
The bug is fixed, and has its own test. This test is no longer needed and will be removed.